_includes/head.html

```diff
@@ -13,6 +13,9 @@
     <!-- Nav infrastructure and misc utils (search, user cards, embeds, CC license) -->
     <link rel="stylesheet" href="{{ site.github.url }}/stylesheets/infrastructure-utils.css" />
 
+    <!-- Python code highlighting -->
```

Comment:
jeykll has built-in support for syntax highlighting. I'm not sure that we need to include this manually.

See http://jekyllrb.com/docs/templates/#code-snippet-highlighting